src/ChromeManagement/GoogleChromeManagementV1HeartbeatStatusReport.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/*
+ * Copyright 2014 Google Inc.
```

Comment:
```suggestion
 * Copyright 2023 Google Inc.
```

Reply:
This shouldn't block this PR, so I added an issue to track it here: https://github.com/googleapis/google-api-php-client-services/issues/2863